src/js/index.js

```diff
@@ -88,8 +88,14 @@ const createBailRaceTable = () => {
   const initSort = 4; // initially sort by difference
   const tableElement = document.getElementById("bail-race-table");
 
-  new RankedTable(BAIL_RACE_DATA, columnConfigs, initSort, tableElement);
+  return new RankedTable(BAIL_RACE_DATA, columnConfigs, initSort, tableElement);
 };
 
-createBailRateTable();
-createBailRaceTable();
+/* RENDER PAGE */
+const bailRateTable = createBailRateTable();
+
+const bailRaceTable = createBailRaceTable();
+document.getElementById("race-outliers").addEventListener("click", (e) => {
+  const showOutliers = bailRaceTable.toggleOutliers();
+  e.target.className = showOutliers ? "outliers-btn showing" : "outliers-btn";
+});
```

Comment:
How would you like to handle these? Leave them outside of the Table class? Create a TableContainer wrapper class, and then the TableContainer would have a reference to the search, outlier, and viewall buttons? Would the TableContainer handle the event listeners?

Reply:
Eh, let's leave them out. It's just five lines. (And only some of the tables have them.)